### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!--
+    Please sign your commit and our DCO.
+
+    Details on how to sign commits can be found at https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits
```
Comment (roidelapluie) [inline .github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md:4]: I think we should not require GPG signing of git commits. That sets the bar too high.

Comment (grobie) [inline .github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md:4]: Do we really want to ask people to GPG sign commits? This is how I understand the sentence now. Isn't that raising the bar for contributions significantly? Unfortunately Github doesn't seem to have docs for DCO signing.

The DCO app instruction page calls it _sign-off_ to differentiate from commit _signing_.

```suggestion
    Please sign-off the Developer Certificate of Origin (DCO): https://developercertificate.org/ 
    You can sign-off commits by adding the -s / --sign-off flag to your git commit.
    More details on how to sign-off commits can be found at https://github.com/apps/dco
```

**Review by grobie — COMMENTED** [review]
Cool in general, just some questions around GPG vs. DCO signing.

File: .github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (grobie) [reply .github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md:4]: Heh, you beat me to it ;-)

Comment (RichiH) [reply .github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md:4]: Due to they way Git works, `git --sign` is valid and results in `--signoff`, which is where this is coming from. But yes, you are correct, thanks :)

**Comment by roidelapluie** [comment]
@grobie can you give your green light? so I can merge https://github.com/prometheus/prometheus/pull/8985/files too.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<!--
+    Please sign CNCF's Developer Certificate of Origin and sign-off your commits by adding the -s / --sign-off flag to `git commit`
+
+    More information on both of those can be found at https://github.com/apps/dco
```
Comment (roidelapluie) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: Don't you think that users would be confused that they need to configure that app?

**Comment by grobie** [comment]
I've been hiking the last week, and still no access to my GitHub account.
Please feel free to merge!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (RichiH) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: I think it's clear enough, but we can always add a new URL if we find a better one.

**Pull request merged**